velox/functions/prestosql/aggregates/SumAggregate.h

```diff
@@ -166,6 +166,13 @@ bool registerSumAggregate(const std::string& name) {
           .intermediateType("double")
           .argumentType("double")
           .build(),
+      exec::AggregateFunctionSignatureBuilder()
+          .argumentType("DECIMAL(a_precision, a_scale)")
+          .intermediateType("DECIMAL(38, i_scale)")
```

Comment:
Looks like we can combine i_scale and r_scale into a single variable and use 'r_scale' in both places. This way, it will be clearer that intermediate and return types are the same.